### DISCUSSION TITLE
Как "скрыть" zapret, чтобы он работал на фоне без окна?

### DISCUSSION THREAD

**Discussion by kiroffYT** [bol-van/zapret#995] [Q&A]
Здравствуйте, скачал zapret по этой ссылке: https://github.com/bol-van/zapret/blob/master/docs/quick_start_windows.md#%D1%81%D0%B0%D0%BC%D0%BE%D0%B5-%D0%BF%D1%80%D0%BE%D1%81%D1%82%D0%BE%D0%B5-%D1%80%D0%B5%D1%88%D0%B5%D0%BD%D0%B8%D0%B5

Софт работает нормально, даже лучше, чем ожидалось - теперь могу смотреть ТикТок без проблем. Однако есть одна небольшая проблема.
Дело в том, что мне хотелось бы **_сделать так, чтобы программа работала в фоне БЕЗ каких-либо окон, короче говоря, чтобы просто был запущенный процесс без UI и окна CMD._** Как это сделать?

**Comment by bol-van** [comment]
Установить как службу
Для этого есть примеры батников service_*cmd, но здесь надо в них разбираться и понимать что куда писать

**Comment by icynstant** [comment]
В планировщик задач запуск при включении пк, будет работать в фоне, без окна

**Comment by Darwin1546** [comment]
Там зачем-то все закомментировано, переусложнено
Насколько я понимаю нужно раскомментировать **set ARGS=^** и **call :srvinst winws1**
И уже потом запускать. Но это конечно странно очень что нужно что-то руками трогать в однокнопочном бандле
